TTS/tts/utils/text/__init__.py

```diff
@@ -28,9 +29,23 @@
 
 
 def text2phone(text, language):
+    '''Convert graphemes to phonemes. For most of the languages, it calls
+    the phonemizer python library that calls espeak/espeak-ng. For chinese
+    mandarin, it calls pypinyin + custom function for phonemizing
+        Parameters:
+                text (str): text to phonemize
+                language (str): language of the text
+        Returns:
+                ph (str): phonemes as a string seperated by "|"
+                        ph = "ɪ|g|ˈ|z|æ|m|p|ə|l"
     '''
-    Convert graphemes to phonemes.
-    '''
+
+    # TO REVIEW : How to have a good implementation for this?
+    if language == "chinese-mandarin":
```

Comment:
maybe you can add the shorthand notation of the language too like ```ch-mn```

Reply:
> Can you rebase the latest dev? Due to #657, some of your CI tests do not pass.

I've done it and pushed it. 

I wanted to know, what is your politic about contributions commits ? Should Pull Requester should squash the different commits of a PR ? Or is it fine otherwise ? (I couldn't find an answer to this on the [contributions guidelines](https://github.com/mozilla/TTS/blob/master/CONTRIBUTING.md)

Reply:
I've replaced it by [google iso code](https://cloud.google.com/translate/docs/languages) for languages.

I wonder how we can do to avoid this if statement at the beginning of text2phone that don't feel scalable for when more languages will appear (with potentially different text2phone mecanics)